var/spack/repos/builtin/packages/py-pylint/package.py

```diff
@@ -33,7 +33,10 @@ class PyPylint(Package):
     version('1.4.1', 'df7c679bdcce5019389038847e4de622')
     version('1.4.3', '5924c1c7ca5ca23647812f5971d0ea44')
 
-    extends('python')
+    extends('python', ignore=r'bin/pytest')
+    depends_on('py-six', type=nolink)
```

Comment:
If we want to take this route, I think we will need to do a sweep of all the packages to do this, so I probably won't do this in this PR.